.github/workflows/verifyPermit2Bytecode.yml

```
@@ -0,0 +1,113 @@
+          if git diff --name-only origin/main...HEAD | grep -q "config/permit2Proxy.json"; then
+            echo "config/permit2Proxy.json has been modified in this branch"
+            echo "CONTINUE=true" >> $GITHUB_ENV
+          else
+            echo "No changes in config/permit2Proxy.json detected in this branch"
+            echo "CONTINUE=false" >> $GITHUB_ENV
+          fi
+
+      - name: Detect New or Changed Permit2 Addresses
+        if: env.CONTINUE == 'true'
+        id: detect-changes
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:20:29: Double quote to prevent globbing and word splitting

(shellcheck)